lib/validators.rb

```diff
@@ -280,9 +280,17 @@ def check_configuration(configuration, periodic_heuristic)
     end
 
     def check_clustering_parameters(configuration)
+      if @hash[:relations].to_a.any?{ |relation| relation[:type] == 'vehicle_trips' }
+        if configuration[:preprocessing][:partitions].to_a.size.positive?
+          raise OptimizerWrapper::DiscordantProblemError.new(
+            'Clustering is not able to deal with vehicle_trips relation for now'
+          )
```

Comment:
[:partitions]&.any? instead of [:partitions].to_a.size.positive? 

'Partitioning (or :partitions parameter) is not currently available with vehicle_trips relation'